### PR TITLE
CrystalThistle!

### PR DESCRIPTION
**Pull request by jimmy12or**
## About the PR
Brings a glass producing plant to botany!

## Why / Balance

For a long time I've wanted to be able to grow glass as a botanist for many projects and departments.

## Technical details

Adds a mutated version of Galaxythistle called Crystalthistle that grows with a new reagent called Quartzite. heating up 10u of quartzite produces a sheet of glass. 
The plant also grows Ethyloxyephedrine which med can use to treat those with narcolepsy.
Quartzite also acts as a weaker version of razorium when drank (it's basically glass shards. should hurt)

## Media

Screenshot 2024-07-02 231013
Screenshot 2024-07-02 231039
Screenshot 2024-07-02 231106
Screenshot 2024-07-02 231226
Screenshot 2024-07-02 231429

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

No breaking changes that I know of.
**Changelog**

<!--
Make sure to take this Changelog template out of the comment block in order for it to show up.
:cl:
- add: Adds Crystalthistle! A plant that produces Quartzite!
- add: Adds Quartzite! A reagent that turns into glass when heated!

### CODE REVIEW

**Comment by jimmy12or** [comment]
Just in case stating it is needed, this PR is ready for review.

File: Resources/Locale/en-US/seeds.ftl [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Seeds
+seeds-crystalthistle-name = crystalthistle
+seeds-crystalthistle-display-name = crystalthistle
```
Comment (deltanedas) [inline Resources/Locale/en-US/seeds.ftl]: put it in deltav folder

File: Resources/Prototypes/DeltaV/Entities/Objects/Specific/Hydroponics/seeds.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+- type: entity
+  parent: SeedBase
+  name: packet of crystalthistle seeds
+  description: "Like a bag of needles"
+  id: crystalthistleSeeds
```
Comment (deltanedas) [inline Resources/Prototypes/DeltaV/Entities/Objects/Specific/Hydroponics/seeds.yml]: use PascalCase

File: Resources/Prototypes/Entities/Markers/Spawners/Random/Food_Drinks/food_produce.yml [hunk 3]
```diff
@@ -59,4 +59,5 @@
       - FoodGalaxythistle
       - FoodFlyAmanita
       - DeathNettle
+      - FoodCrystalthistle
```
Comment (deltanedas) [inline Resources/Prototypes/Entities/Markers/Spawners/Random/Food_Drinks/food_produce.yml]: add deltav comment

File: Resources/Prototypes/Hydroponics/seeds.yml [hunk 4]
```diff
@@ -1192,6 +1192,8 @@
   packetPrototype: GalaxythistleSeeds
   productPrototypes:
     - FoodGalaxythistle
+  mutationPrototypes:
```
Comment (deltanedas) [inline Resources/Prototypes/Hydroponics/seeds.yml:1195]: .

File: Resources/Textures/Objects/Specific/Hydroponics/crystal_thistle.rsi/meta.json [hunk 5]
```diff
@@ -0,0 +1,32 @@
+{
```
Comment (deltanedas) [inline Resources/Textures/Objects/Specific/Hydroponics/crystal_thistle.rsi/meta.json]: rsi should be in DeltaV folder, then update the yml

**Review by deltanedas — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Hydroponics/seeds.yml (hunk 4 above)
Comment (jimmy12or) [reply Resources/Prototypes/Hydroponics/seeds.yml:1195]: ?

Comment (Timemaster99) [reply Resources/Prototypes/Hydroponics/seeds.yml:1195]: Add DeltaV comment

File: Resources/Locale/en-US/seeds.ftl (hunk 1 above)
Comment (jimmy12or) [reply Resources/Locale/en-US/seeds.ftl]: File moved

File: Resources/Prototypes/DeltaV/Entities/Objects/Specific/Hydroponics/seeds.yml (hunk 2 above)
Comment (jimmy12or) [reply Resources/Prototypes/DeltaV/Entities/Objects/Specific/Hydroponics/seeds.yml]: corrected in all instances I've added

File: Resources/Prototypes/Entities/Markers/Spawners/Random/Food_Drinks/food_produce.yml (hunk 3 above)
Comment (jimmy12or) [reply Resources/Prototypes/Entities/Markers/Spawners/Random/Food_Drinks/food_produce.yml]: Done~

File: Resources/Prototypes/Hydroponics/seeds.yml (hunk 4 above)
Comment (jimmy12or) [reply Resources/Prototypes/Hydroponics/seeds.yml:1195]: Done~

File: Resources/Textures/Objects/Specific/Hydroponics/crystal_thistle.rsi/meta.json (hunk 5 above)
Comment (jimmy12or) [reply Resources/Textures/Objects/Specific/Hydroponics/crystal_thistle.rsi/meta.json]: Done and tested!

**Review by deltanedas — APPROVED** [review]

**Pull request merged**